src/main.cpp

```diff
@@ -1,4 +1,5 @@
 #include <Arduino.h>
+#include "analog.h"
 
 // put function declarations here:
 int myFunction(int, int);
```

Comment:
Let's add a call to `initAnalog()` in `setup()` and also remove the template `myFunction()`